Add coverage for content host rename procedure(BZ1417953)

oshtaier opened the pull request:
```
nosetests tests/foreman/ui/test_hostunification.py -m test_positive_rename_content_host
..
----------------------------------------------------------------------
Ran 2 tests in 311.316s

OK
```

ldjebran commented on tests/foreman/ui/test_hostunification.py:
```diff
@@ -256,6 +257,39 @@ def test_positive_rename_content_host(self):
+                    new_name=new_name,
+                )
+                self.assertIsNotNone(self.contenthost.search(new_name))
+            self.assertIsNotNone(Host.info({'name': new_name}))
```
if host with new_name does not exist we will have an CLIReturnCodeError
exception and not a None Value, think more appropriate to use with self.assertNotRaises(CLIReturnCodeError) and to compare the host by id eg: get the host by name at the beginning and create a new host object inside "with self.assertNotRaises" and compare that the ids are identical

ldjebran replied:
of get the id by the old name at the beginning, and get info at the end by id and check that the name is equal to new_name

oshtaier commented:
@ldjebran @svtkachenko Addressed comments
```
 nosetests tests/foreman/ui/test_hostunification.py -m test_positive_rename_content_host_cli
.
----------------------------------------------------------------------
Ran 1 test in 142.106s

OK
```
